statsmodels/genmod/generalized_linear_model.py

```diff
@@ -40,6 +40,7 @@
 
 from statsmodels.tools.sm_exceptions import (PerfectSeparationError,
                                              DomainWarning)
+from cvxopt.base import matrix
```

Comment:
how did this get in here? unit tests fail
That's the second time in this PR that I get an unrelated import added to a module.
I'm only editing the statsmodels source in pydev/Eclipse.